packages/autorest.typescript/test/integration/generated/additionalProperties/src/additionalPropertiesClient.ts

```diff
@@ -1,10 +1,8 @@
-/*
- * Copyright (c) Microsoft Corporation.
- * Licensed under the MIT License.
- *
- * Code generated by Microsoft (R) AutoRest Code Generator.
- * Changes may cause incorrect behavior and will be lost if the code is regenerated.
- */
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Code generated by Microsoft (R) AutoRest Code Generator.
```

Comment:
why duplicted?

Reply:
According to the above comment, the src folder uses 
```
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

// Code generated by Microsoft (R) AutoRest Code Generator.
// Changes may cause incorrect behavior and will be lost if the code is regenerated.
```
and Tests and samples use 
```
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
```